target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts

```diff
@@ -0,0 +1,379 @@
+			      &pmx_led_usb2_green &pmx_led_usb2_red
+			      >;
+		pinctrl-names = "default";
+
+		led_status_green: status_green {
+			label = "green:status";
+			gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_red: status_red {
+			label = "red:status";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+		};
+
+		cloud_blue {
+			label = "blue:cloud";
+			gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
+		};
+
+		disk1_green {
+			label = "green:disk1";
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata1";
+		};
+
+		disk1_red {
+			label = "red:disk1";
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+		};
+
+		disk2_green {
+			label = "green:disk2";
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata2";
+		};
+
+		disk2_red {
+			label = "red:disk2";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+
+		disk_fail_green {
+			label = "green:disk-fail";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+
+		disk_fail_red {
+			label = "red:disk-fail";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1_green {
+			label = "green:usb1";
+			gpios = <&gpio1 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port2>;
+		};
+
+		usb1_red {
+			label = "red:usb1";
+			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2_green {
+			label = "green:usb2";
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port1>;
+		};
+
+		usb2_red {
+			label = "red:usb2";
+			gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&eth0 {
+	status = "okay";
+};
+
+&eth0port {
+	phy-handle = <&ethphy9>;
+};
+
+&i2c0 {
+	status = "okay";
+
+	rtc@30 {
+		compatible = "s35390a";
+		reg = <0x30>;
+	};
+
+	lm63@4c {
+		compatible = "national,lm63";
+		reg = <0x4c>;
+	};
+};
+
+&mdio {
+	status = "okay";
+	ethphy9: ethernet-phy@9 {
+		reg = <9>;
+	};
+};
+
+&nand {
+	status = "okay";
+	chip-delay = <40>;
+
+	partition@0 {
+		label = "uboot";
+		reg = <0x0000000 0x200000>;
+	};
+	partition@200000 {
+		label = "certificate";
+		reg = <0x0200000 0x100000>;
+	};
+	partition@300000 {
+		label = "preset_cfg";
+		reg = <0x0300000 0x100000>;
+	};
+	partition@400000 {
+		label = "dev_params";
+		reg = <0x0400000 0x100000>;
+	};
+	partition@500000 {
+		label = "active_bank";
+		reg = <0x0500000 0x0100000>;
+	};
+	partition@600000 {
+		label = "magic";
+		reg = <0x0600000 0x0100000>;
+	};
+	partition@700000 {
+		label = "bank1";
+		reg = <0x0700000 0x2800000>;
+	};
+	partition@2f00000 {
+		label = "bank2";
+		reg = <0x2f00000 0x2800000>;
+	};
+	partition@5a00000 {
+		label = "reserved";
+		reg = <0x5a00000 0x2000000>;
+	};
+	partition@7a00000 {
+		label = "ubi";
+		reg = <0x7a00000 0x8600000>;
+	};
+};
+
+&pciec {
+	status = "okay";
+};
+
+&pcie0 {
+	status = "okay";
+};
+
+&pinctrl {
+	/* buzzer1 and buzzer2 gpios are connected to two pins of buzzer
+	 * leave it as is due lack of proper driver
+	 */
+	pmx_buzzer1: pmx-buzzer1 {
+		marvell,pins = "mpp12";
```

Comment:
Could you consider consolidating the pinmux nodes? (like grouping into leds/buttons/buzzer nodes)
https://github.com/openwrt/openwrt/blob/master/target/linux/mvebu/files/arch/arm/boot/dts/armada-370-buffalo-ls421de.dts#L434

Reply:
It's much simpler. Done.